src/NetworkTest/types/opentok/rtcStats.ts

```diff
@@ -0,0 +1,159 @@
+export interface RTCStatsInternal {
```

Comment:
opentok.d.ts references RTCStatsReport without including it. I think is was added (as a standard) in some version of typescript. (It should be in 3.7.7.)

Reply:
Correct, @jeffswartz. I just wanted to confirm whether updating the TypeScript version. Also, I have to double-check, but I believe there were some changes made to the interface in more recent versions of TypeScript that we should be aware of.
-> If not I will include that changes in the bundle branch